### DISCUSSION TITLE
Atom has become extremely slow

### DISCUSSION THREAD

**Discussion by Putnik1** [atom/atom#22557] [Q&A]
I am using the latest Atom 1.57.0 on macOS Catalina,
but recently it has become incredibly slow (having to wait nearly a minute before it finishes opening ("beachball" showing).

I have about 50 tabs across 3 windows.  And I had noticed there were also around 10 completely blank windows that it had somehow created (but I managed to close them).

Is this a known problem, and is there a fix for it please?
Many thanks!

**Comment by ThatXliner** [comment]
Have you tried opening 50 tabs on various websites (maybe something memory-hoggy) across 3 windows on Google Chrome, quit the app (not closing the window), and try to open it again? As it restores the tabs, try to count the time it takes to load all the websites again.

Atom uses Electron, which uses Chromium. Atom *may* be a little slower due to the nature of it's package system and the init script and all, so you may need to wait.

**Reply by ThatXliner** [reply]
For known issues, I wonder if #13253 could help
